.stylelintrc.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
- [ ] [OPTIONAL] It is best practice that your commit messages be consistent in the capitalization of the first letter and the style. Kindly take note going forward on other projects. Please do not bother about editing them now.